### ISSUE TITLE
java.security.cert.CertPathValidatorException: Trust anchor for certification path not found.

### ISSUE THREAD

**Issue by cyril23** [Pushwoosh/pushwoosh-phonegap-plugin#333]
Hello guys,

today on 2020-06-04 11:38:10.903 (UTC+2), a pushwoosh registration failed with the following message:

`"java.security.cert.CertPathValidatorException: Trust anchor for certification path not found."`

The client phone is a Samsung Galaxy Grand Prime `SM G530FZ` with `And 5.0.2`.

We did not have this problem on any other phones so far.

Do you have any idea?

phonegap config:
```
<preference name="phonegap-version" value="cli-8.0.0" />
<preference name="KeepRunning" value="true"/>
<preference name="AndroidLaunchMode" value="singleInstance"/>
<preference name="android-minSdkVersion" value="19" />	
<preference name="android-targetSdkVersion" value="26" /> 
<plugin name="pushwoosh-pgb-plugin" spec="7.11.0" />
```

**Comment by wfhm** [comment]
@cyril23,

Could you please post a full exception message here?

**Comment by cyril23** [comment]
> @cyril23,
> 
> Could you please post a full exception message here?

I have not logged it. 

However today 2020-06-04 18:01:35.642 (UTC+2), the push registration process was successful - on the same device. Very strange...

**Comment by wfhm** [comment]
@cyril23 Indeed, really strange. The issue should be either something with the recent [AddTrust CA Root expiry](https://support.sectigo.com/articles/Knowledge/Sectigo-AddTrust-External-CA-Root-Expiring-May-30-2020), or it could be related to older TLS versions not being supported on an old Android version, however, it does not explain why it disappeared. Please let us know if the issue appears again, and it would be really helpful if you could get the full exception log.
